src/frontend/components/component-info/component-info.ts

```diff
@@ -32,6 +32,23 @@ export default class ComponentInfo {
     }
   }
 
+  viewComponentSource($event) {
+    const highlightStr = '[batarangle-id=\"' + this.node.id + '\"]';
+
+    let evalStr = `inspect(ng.probe(document.querySelector('${highlightStr}'))
+    .componentInstance.constructor)`;
+
+    chrome.devtools.inspectedWindow.eval(
+      evalStr,
+      function(result, isException) {
+        console.log(result, isException);
```

Comment:
This opens source panel for the file we don't want to do anything just display response error or result